packages/site/src/loaders/theme.server.ts

```diff
@@ -1,11 +1,12 @@
 import { createCookieSessionStorage, json } from '@remix-run/node';
-import { isTheme, Theme } from '@myst-theme/providers';
+import { isTheme } from '@myst-theme/providers';
+import type { Theme } from '@myst-theme/providers';
 import type { ActionFunction } from '@remix-run/node';
 
 export const themeStorage = createCookieSessionStorage({
   cookie: {
     name: 'theme',
-    secure: true,
+    secure: false,
```

Comment:
What is the reason for this change?

Reply:
For me (Firefox), a local MyST http server can't use cookies without disabling this.

Reply:
Can we pop this in another PR/issue? I can handle it in my testing pass.